blueflood-http/src/test/java/com/rackspacecloud/blueflood/inputs/formats/JSONMetricsContainerTest.java

```diff
@@ -53,7 +53,7 @@ public void testJSONMetricsContainerConstruction() throws Exception {
         Assert.assertEquals(1234566, metricsCollection.get(0).getTtlInSeconds());
         Assert.assertEquals(1234567890L, metricsCollection.get(0).getCollectionTime());
         Assert.assertEquals("milliseconds", metricsCollection.get(0).getUnit());
-        Assert.assertEquals("L", metricsCollection.get(0).getDataType().toString());
+        Assert.assertEquals("N", metricsCollection.get(0).getDataType().toString());
```

Comment:
No one uses this data type marker, I think. And the type of the data can be easily referred from the JSON types. But its obviously beyond of the scope of this PR, but we should do it eventually.